jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/AssetFileGenerator.java

```diff
@@ -211,12 +213,16 @@ public void generateFiles(List<SubscriptionVH> subscriptions, String filePath) {
 					.fetchPolicyDefinitionDetails(subscription);
 			ExecutorService executor = Executors.newCachedThreadPool();
 
+			List<String> targetTypeList = new CopyOnWriteArrayList<>();
+
 			executor.execute(() -> {
 				if (!(isTypeInScope("virtualmachine"))) {
 					return;
 				}
 				try {
+					targetTypeList.add("virtualmachine");
 					FileManager.generateVMFiles(vmInventoryCollector.fetchVMDetails(subscription, tagMap));
+					targetTypeList.remove("virtualmachine");
```

Comment:
I see 3 magic strings here - please make it as a constant for this class or module. same for all other target types.
it will be less of a hustle to maintain.

Reply:
okay @mkinchin-paladin

Reply:
done.